packages/oui-themes/README.md

```diff
@@ -0,0 +1,39 @@
+# Themes
+
+<component-status cx-design="partial" ux="rc"></component-status>
+
+oui-themes is a package which provides already built themes that be used for different scenarios.
+
+## Installation
+
+```less
+@import 'oui-themes/themes';
+```
+
+## Usage
+
+### Sapphire
+
+This theme is intended to be mainly used by the OVH Manager.
```

Comment:
✏️  Could you please replace `Manager` by `Control Panel`?

Reply:
Done